service/src/main/kotlin/fi/espoo/evaka/varda/new/VardaQueries.kt

```diff
@@ -176,3 +178,72 @@ fun Database.Read.getVardaPerson(childId: ChildId): VardaPerson =
             )
         }
         .exactlyOne()
+
+fun Database.Transaction.addNewChildrenForVardaUpdate(migrationSpeed: Int = 0) {
+    if (migrationSpeed > 0) {
+        // Move children from varda_reset_child (old integration) to varda_state (new integration)
+        createUpdate {
+                sql(
+                    """
+                    INSERT INTO varda_state (child_id, state)
+                    SELECT evaka_child_id, null
+                    FROM varda_reset_child
+                    WHERE NOT EXISTS (SELECT FROM varda_state WHERE child_id = evaka_child_id)
+                    LIMIT ${bind(migrationSpeed)}
+                    """
+                )
+            }
+            .execute()
```

Comment:
Poista myös `varda_reset_child` -taulusta jotta vanha integraatio ei käsittele näitä lapsia enää.